File: WebViewApp/app/src/main/java/com/example/webviewapp/contract/SignupContract.java
package com.example.webviewapp.contract;

import android.text.Editable;

import com.example.webviewapp.data.User;

import java.util.List;

public interface SignupContract {

    interface View {
    }

    interface Presenter {
        void SignUp();

        Boolean ContainsUid(Editable uidtext);

        List<User> getUser();
    }
}

File: WebViewApp/app/src/main/java/com/example/webviewapp/contract/SignUpContract.kt
package com.example.webviewapp.contract

import android.text.Editable
import com.example.webviewapp.data.User

interface SignUpContract {
    interface View
    interface Presenter {
        fun signUp()
        fun containsUid(uidText: Editable?): Boolean?
        val user: List<User?>?
    }
}